evaluate.py

```diff
@@ -132,23 +130,38 @@ def run_test(init_op, dataset):
 
             bar.finish()
 
+            # Print test summary
             wer, cer, samples = calculate_report(wav_filenames, ground_truths, predictions, losses)
             mean_loss = np.mean(losses)
+            print('Test on %s - WER: %f, CER: %f, loss: %f' % (dataset, wer, cer, mean_loss))
+            print('-' * 80)
 
-            # Take only the first report_count items
-            report_samples = itertools.islice(samples, FLAGS.report_count)
+            # Take only the first, median and last report_count items
+            best_samples = samples[:FLAGS.report_count]
```

Comment:
I'm unsure about the differences here with `itertools.islice()` wrt your way to do it.

Reply:
Not much here, but it allows negative indexing for the worst samples.

Reply:
I'd worry about performances differences, that's all.

Reply:
According to this question: https://stackoverflow.com/q/2738096
it should be faster than islice(). i think its also more readable.

Reply:
Yeah, but it's not unlikely that we had that for a reason, and I can't recall why. maybe @reuben remembers?